ia/src/process.py

```
@@ -1,8 +1,8 @@
 import sys
-from models import iaParser
-from parser import ia_parser
-from utils import exit_error
-# from message import *
+# from models import iaParser
+# from parser import ia_parser
+# from utils import exit_error
+from message import *
```

Ruff (0.11.9)

`from message import *` used; unable to detect undefined names

(F403)